[2025-04-17T03:31:21.106+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-17T03:31:21.157+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: plusvalia_pipeline.extract_geodata manual__2025-04-17T03:29:21.105577+00:00 [queued]>
[2025-04-17T03:31:21.173+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: plusvalia_pipeline.extract_geodata manual__2025-04-17T03:29:21.105577+00:00 [queued]>
[2025-04-17T03:31:21.175+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-17T03:31:21.197+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_geodata> on 2025-04-17 03:29:21.105577+00:00
[2025-04-17T03:31:21.210+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'plusvalia_pipeline', 'extract_geodata', 'manual__2025-04-17T03:29:21.105577+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/plusvalia_pipeline_dag.py', '--cfg-path', '/tmp/tmpnew9sz94']
[2025-04-17T03:31:21.214+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask extract_geodata
[2025-04-17T03:31:21.215+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=277) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-17T03:31:21.217+0000] {standard_task_runner.py:63} INFO - Started process 279 to run task
[2025-04-17T03:31:21.290+0000] {task_command.py:426} INFO - Running <TaskInstance: plusvalia_pipeline.extract_geodata manual__2025-04-17T03:29:21.105577+00:00 [running]> on host ee6c3cbd6aa6
[2025-04-17T03:31:21.454+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Brayan Miranda' AIRFLOW_CTX_DAG_ID='plusvalia_pipeline' AIRFLOW_CTX_TASK_ID='extract_geodata' AIRFLOW_CTX_EXECUTION_DATE='2025-04-17T03:29:21.105577+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-17T03:29:21.105577+00:00'
[2025-04-17T03:31:21.456+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-17T03:31:21.474+0000] {logging_mixin.py:188} INFO - Cargando shapefiles y servicios...
[2025-04-17T03:31:24.439+0000] {raw.py:723} INFO - Created 2,345 records
[2025-04-17T03:31:24.863+0000] {raw.py:723} INFO - Created 92 records
[2025-04-17T03:31:25.037+0000] {raw.py:723} INFO - Created 184 records
[2025-04-17T03:31:25.070+0000] {logging_mixin.py:188} INFO - Datos geoespaciales cargados y guardados en data/processed/
[2025-04-17T03:31:25.078+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-17T03:31:25.080+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-17T03:31:25.096+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=plusvalia_pipeline, task_id=extract_geodata, run_id=manual__2025-04-17T03:29:21.105577+00:00, execution_date=20250417T032921, start_date=20250417T033121, end_date=20250417T033125
[2025-04-17T03:31:25.128+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-17T03:31:25.166+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-17T03:31:25.186+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
